bears/general/KeywordBear.py

```diff
@@ -79,22 +79,54 @@ def run(self,
             filename,
             file,
             keywords: list=['todo', 'fixme'],
+            regex_keyword: str='',
             dependency_results: dict=None):
         '''
         Checks the code files for given keywords.
 
         :param keywords:
             A list of keywords to search for (case insensitive).
             Default are TODO and FIXME.
+        :param regex_keyword:
+            A regular expression to search for matching keywords in a file.
         '''
         keywords_regex = re.compile(
             '(' + '|'.join(re.escape(key) for key in keywords) + ')',
             re.IGNORECASE)
 
+        regex = re.compile(regex_keyword)
+
         comments = _get_comments(dependency_results)
 
+        message = "The line contains the keyword '{}'."
+        yield from self.check_keywords(filename, file, comments,
+                                       keywords_regex, message)
+
+        if regex_keyword is not '':
+            message = ("The line contains the keyword '{}' which "
+                       'resulted in a match with given regex.')
+            yield from self.check_keywords(filename, file, comments, regex,
+                                           message)
+
+    def check_keywords(self,
+                       filename,
+                       file,
+                       comments,
+                       regex,
+                       message):
+        '''
+        Checks for the presence of keywords according to regex in a given file.
+
+        :param regex:
+           A regular expression which is used to search matching
+           keywords in a file.
+        :param message:
+           A message to be displayed to the user when a keyword in a given file
+           results in a match.
```

Comment:
@sils I didn't get it... are you referring to '{}' in message text which gets replaced by actual keyword ?
the modification that is needed is to modify :param message: so that it also has the line 'it may have an unnamed placeholder for the keyword' 
Is that all  or something else is needed ?